### PR TITLE
[Rule Tuning] Suspicious PrintSpooler Service Executable File Creation

### PR DESCRIPTION
**Pull request by w0rk3r**
## Summary

Converts the rule to the new_terms rule type, uses query DSL to exclude noisy patterns, and to include the `system32` & `syswow64` folders.

The rule is very noisy and the majority of the hits are FPs, so this aims to reduce the volume and improve the FP x TP rate.

image

### CODE REVIEW

**Comment by protectionsmachine** [comment]
## Rule: Tuning - Guidelines

These guidelines serve as a reminder set of considerations when tuning an existing rule.

### Documentation and Context

- [ ] Detailed description of the suggested changes.
- [ ] Provide example JSON data or screenshots.
- [ ] Provide evidence of reducing benign events mistakenly identified as threats (False Positives).
- [ ] Provide evidence of enhancing detection of true threats that were previously missed (False Negatives).
- [ ] Provide evidence of optimizing resource consumption and execution time of detection rules (Performance).
- [ ] Provide evidence of specific environment factors influencing customized rule tuning (Contextual Tuning).
- [ ] Provide evidence of improvements made by modifying sensitivity by changing alert triggering thresholds (Threshold Adjustments).
- [ ] Provide evidence of refining rules to better detect deviations from typical behavior (Behavioral Tuning).
- [ ] Provide evidence of improvements of adjusting rules based on time-based patterns (Temporal Tuning).
- [ ] Provide reasoning of adjusting priority or severity levels of alerts (Severity Tuning).
- [ ] Provide evidence of improving quality integrity of our data used by detection rules (Data Quality).
- [ ] Ensure the tuning includes necessary updates to the release documentation and versioning.

### Rule Metadata Checks

- [ ] `updated_date` matches the date of tuning PR merged.
- [ ] `min_stack_version` should support the widest stack versions.
- [ ] `name` and `description` should be descriptive and not include typos.
- [ ] `query` should be inclusive, not overly exclusive. Review to ensure the original intent of the rule is maintained.

### Testing and Validation

- [ ] Validate that the tuned rule's performance is satisfactory and does not negatively impact the stack.
- [ ] Ensure that the tuned rule has a low false positive rate.

File: rules/windows/privilege_escalation_printspooler_service_suspicious_file.toml [hunk 1]
```diff
@@ -42,24 +34,61 @@ tags = [
...
+event.category : "file" and host.os.type : "windows" and event.type : "creation" and
+  process.name : "spoolsv.exe" and file.extension : "dll"
 '''
 
+[[rule.filters]]
+[rule.filters.meta]
+negate = false
+[rule.filters.query.wildcard."file.path"]
+"case_insensitive" = true
+"value" = "?:\\\\Windows\\\\Sys?????\\\\*"
```
Comment (w0rk3r) [inline rules/windows/privilege_escalation_printspooler_service_suspicious_file.toml:49]: To match both system32 and syswow64

Comment (Aegrah) [reply rules/windows/privilege_escalation_printspooler_service_suspicious_file.toml:49]: Magic

File: rules/windows/privilege_escalation_printspooler_service_suspicious_file.toml [hunk 2]
```diff
@@ -74,3 +103,9 @@ id = "TA0004"
 name = "Privilege Escalation"
 reference = "https://attack.mitre.org/tactics/TA0004/"
 
+[rule.new_terms]
+field = "new_terms_fields"
+value = ["host.id", "file.path"]
```
Comment (Aegrah) [inline rules/windows/privilege_escalation_printspooler_service_suspicious_file.toml:108]: Wouldn't it be beneficial to remove host.id here, as I can assume this rule being quite noisy, and having other hosts to reduce noise would be useful?

**Review by Aegrah — APPROVED** [review]

**Review by terrancedejesus — APPROVED** [review]

File: rules/windows/privilege_escalation_printspooler_service_suspicious_file.toml (hunk 2 above)
Comment (w0rk3r) [reply rules/windows/privilege_escalation_printspooler_service_suspicious_file.toml:108]: I'll try it with the host first and make it broader if needed

**Pull request merged**